docs/testing/webdriverio/01-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WebdriverIO Overview
+sidebar_label: Overview
+---
+
+# Overview
+
+WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and stable test suite.
+
+Testing with WebdriverIO has the following advantages:
+
+- __Cross Browser Support__: WebdriverIO is designed to support all platforms, either on desktop or mobile. You can run tests on actual browser your users are using, including covering different versions of them.
+- __Real User Interaction__: Interaction with elements in WebdriverIO through the WebDriver protocol is much closer to native user-triggered interactions than what can be achieved with emulated DOM environments (such as JSDom or Stencil's own Mock-Doc).
+- __Web Platform Support__: Running tests in actual browsers allows you to tap into the latest Web Platform features for testing your components, often not available when using virtual DOM environments.
+
+## Set Up
+
+To get started with WebdriverIO, all you need to do is to run their project starter:
+
+```sh
+npm init wdio@latest .
+```
```

Comment:
We should leverage the `npm2yarn` plugin. 

NOTE: Github suggestions do no like docusaurus code blocks within them so don't directly commit this since formatting will be off

```suggestion
```bash npm2yarn
npm init wdio@latest .
```
```

Reply:
Yes, this has bugged me soo many times until I learned that you can add additional backticks, e.g.
`````
````suggestion
```bash npm2yarn
npm init wdio@latest .
```
````
`````

Which makes it render correctly 🙌

Reply:
oh hot damn, that is awesome to know!